src/pds_doi_service/core/outputs/web_parser.py

```diff
@@ -22,10 +23,11 @@ class DOIWebParser:
     Abstract base class for parsers of DOI labels returned (or submitted)
     to a DOI service endpoint.
     """
-    _optional_fields = []
+
+    _optional_fields: List[Any] = []  # 🤔 TODO: should be ``List[str]``?
     """The optional Doi field names parsed from labels."""
 
-    _mandatory_fields = []
+    _mandatory_fields: List[Any] = []  # 🤔 TODO: should be ``List[str]``?
```

Comment:
TODO is correct on both counts

Reply:
Oo, on top of this, it should be
```python
_mandatory_fields: list[str] = []
```
Apparently I missed an instance of older `typing.List` when we can just use `list`.